tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c

```diff
@@ -45,15 +45,17 @@ void harness() {
   IotHttpsRequestHandle_t reqHandle = allocate_IotRequestHandle();
   if (reqHandle)
     __CPROVER_assume(is_valid_IotRequestHandle(reqHandle));
-  uint32_t nameLen;
-  uint32_t valueLen;
+
+  size_t nameLen;
+  size_t valueLen;
   __CPROVER_assume(nameLen < CBMC_MAX_OBJECT_SIZE);
   __CPROVER_assume(valueLen < CBMC_MAX_OBJECT_SIZE);
-  char * pName = safeMalloc(nameLen+1);
-  char * pValue = safeMalloc(valueLen+1);
+  char *pName = safeMalloc(nameLen+1);
+  char *pValue = safeMalloc(valueLen+1);
```

Comment:
Is this consistent with the team's uncrustify rules?

Reply:
I haven't checked the uncrustify rules, but this change makes it consistent with the other harnesses, so I'm hoping it is ...